src/models/files/ZosNetworkFile.js

```diff
@@ -67,6 +67,10 @@ export default class ZosNetworkFile {
     return this.data.proxies || {}
   }
 
+  get nonUpgradeableInstances() {
+    return this.data.nonUpgradeableInstances || {}
```

Comment:
Just throwing it out there: how about renaming `proxies` to `instances` (yes, it is a breaking change), and adding an `upgradeable` flag on each of them? That would also help by removing the mention of proxies from the json file.

Reply:
I started doing exactly what you suggest, but it was a heavy braking change so I decided to stop to discuss it with you guys before moving forward.